src/SwedbankPay.Sdk.Tests/TestBuilders/PaymentRequestBuilder.cs

```diff
@@ -123,7 +116,7 @@ public Payments.TrustlyPayments.TrustlyPaymentRequest BuildTrustlyRequest()
                 this.trustlyPrefillInfo);
         }
 
-        public PaymentRequestBuilder WithCreditcardTestValues(Guid payeeId, Operation operation = null, Intent intent = Intent.Authorization)
+        public PaymentRequestBuilder WithCreditcardTestValues(Guid payeeId, Operation operation = null, PaymentIntent intent = PaymentIntent.Authorization)

```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)